inflammation/models.py

```diff
@@ -31,3 +31,14 @@ def daily_max(data):
 def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
+
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    number = np.mean(data, axis=0)
```

Comment:
I don't immediately know what the purpose of the `number` variable is from its name. Based on its assignment, it is the mean along one axis, so perhaps it should be called `data_mean` or event just `mean`?

Reply:
Resolved by using `np.std`